web/src/installers/index.ts

```diff
@@ -233,6 +234,7 @@ export const minioConfigSchema = {
     s3Override: { type: "string", flag: "s3-override", description: "Override the download location for addon package distribution (used for CI/CD testing alpha addons)" },
     namespace: { type: "string", flag: "minio-namespace", description: "The namespace Minio is installed to" },
     hostPath: { type: "string", flag: "minio-hostpath", description: "Configure the minio deployment to use a local hostPath for storing data." },
+    migrateFromRGW: { type: "boolean" },
```

Comment:
Should we add a flag and description for this?

Reply:
I thought they weren't used anywhere

Reply:
but yeah, I'll add

Reply:
I thought it was used to generate the tables on kurl.sh? I might be wrong though